### ISSUE TITLE
[feature]: Developer documentation in other languages than English

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#235]
### Is your feature request related to a problem? Please describe.

I think it would be a great idea to have a small collection of developer documentation in other languages than English.

A quick review of our stars, forks, etc. shows people mostly from India.  So likely those primary and secondary/dialectical  languages used in India should get translated first.

People could add/contribute their own as time goes on.

I would start with out CONTRIBUTING.md 

### Describe the solution you would like

The most helpful developer document we have is CONTRIBUTING.md in the /docs folder.
Translations could be done as CONTRIBUTING.(language).md such as CONTRIBUTING.hindi.md

Some languages could include:

Urdu
Hindi
Odia
Tamil
Gujarati
Duala
Maltese
Mandarin Chinese
Spanish
Arabic
Bengali
Portuguese
Russian
Japanese
Punjabi
French
German
Assamese
Tagalog
Berber
Kswahili / Swahili
Manipuri / Meitei
Malay

### Describe the alternatives you have tried

Some popular projects do include developer documentation and some primary instructions and other documenation  in other languages.  This may reduce the barrier to entry for non native / ESL speakers.

After a quick review in Google Translate, this should be pretty easily done.

**Comment by martinyis** [comment]
Hey, it looks like an interesting issue. Could you assign it to me? I know personally Russian and Spanish

**Comment by gbowne1** [comment]
Ok.  I can assign anyone.

**Comment by akshaya-hub** [comment]
@gbowne1 can I work on this? i know personally Hindi, Telugu

**Comment by gbowne1** [comment]
@akshaya-hub Hindi is in the current PR, but you can definitely add Telugu.

**Comment by akshaya-hub** [comment]
@gbowne1  should i need to translate only the contributing.md file to TELUGU

**Comment by gbowne1** [comment]
Yes and it would then become CONTRIBUTING.telugu.md

@akshaya-hub

**Comment by akshaya-hub** [comment]
@gbowne1  sure

**Comment by akshaya-hub** [comment]
@gbowne1 opened the PR please review it

**Comment by LuckyTigerr** [comment]
@gbowne1, hello, can I make documentation in Russian?

**Comment by gbowne1** [comment]
Sure @LuckyTigerr

**Comment by akshayamadhuri** [comment]
@gbowne1  can i contribute to Punjabi and Bengali

**Comment by gbowne1** [comment]
Yes you can @akshayamadhuri 

The filenames would be CONTRIBUTING.punjabi.md and CONTRIBUTING.bengali.md

**Comment by DavidGomes77** [comment]
@gbowne1 Hey, I could do the Contributing for German, French and Portuguese. Just assign it to me.

**Comment by gbowne1** [comment]
Ok @DavidGomes77

**Comment by laurittab** [comment]
Hi all, I'll work on CONTRIBUTING.french.md as I haven't seen it added so far.
